geom.go

```diff
@@ -164,8 +164,8 @@ func (r *Rect) String() string {
 // NewRect constructs and returns a pointer to a Rect given a corner point and
 // the lengths of each dimension.  The point p should be the most-negative point
 // on the rectangle (in every dimension) and every length should be positive.
-func NewRect(p Point, lengths []float64) (r *Rect, err error) {
-	r = new(Rect)
+func NewRect(p Point, lengths []float64) (r Rect, err error) {
+	// r = new(Rect)
```

Comment:
suggestion: just remove this instead of commenting it out

Reply:
Done